ocaml/doc/README.md

```diff
@@ -0,0 +1,11 @@
+# A note on generating locally the API reference
+
+Run `make doc`. The output is in `_build/ocaml/doc/markdown`.
+
+To view the markdown properly you will need to install `graphviz` so as to
+convert the dot files to images. To generate a pdf in addition, you will need to
+install `pandoc`.
```

Comment:
I think you still have to mention `pdflatex` because you are requiring the latex engine to be `pdflatex` in `ocaml/doc/pandoc.sh`, so `pandoc` cannot fall back to any available engine. Then whomever wants to compile the sdk can look at which latex distribution to install in their system to make sure that the right executable ends up in the path.